openapi/errors.go

```diff
@@ -75,4 +75,6 @@ var TransientErrorRegexes = []*regexp.Regexp{
 	regexp.MustCompile(`Unknown worker environment`),
 	regexp.MustCompile(`ClusterNotReadyException`),
 	regexp.MustCompile(`worker env .* not found`),
+	regexp.MustCompile(`Timed out after `),
```

Comment:
There is space after `after`, is it expected?

Reply:
yes - to show that there is an argument after it